<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyHAT Point Spectra GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>807</width>
         <height>295</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>150</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Console window&lt;br/&gt;This window gives you information about your running module&lt;br/&gt;Errors will also show up here, if they occur&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Your current progression.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="format">
       <string>%p%</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="progress_OK">
      <item>
       <widget class="QPushButton" name="refreshModulePushButton">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;
&lt;p&gt;Refresh the modules&lt;/p&gt;
&lt;p&gt;This will update all the modules so they contain relevant information from previous modules.
&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="undoModulePushButton">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Re-run your last run module&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Re-run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopPushButton">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Completely stop the currently running module&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="insertModuleComboBox">
        <property name="minimumSize">
         <size>
          <width>86</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Insert After</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="deleteModuleComboBox">
        <property name="minimumSize">
         <size>
          <width>86</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Delete</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="okPushButton">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Press this button when you are ready to run the modules in your workflow&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="whatsThis">
         <string>Press this button when you're ready to run. (Ctrl+Enter)</string>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Workflow</string>
    </property>
    <addaction name="actionRestore_Workflow"/>
    <addaction name="actionSave_Current_Workflow"/>
    <addaction name="actionCreate_New_Workflow"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPreprocessing">
    <property name="title">
     <string>Preprocessing</string>
    </property>
    <addaction name="actionResample"/>
    <addaction name="actionRemove_Baseline"/>
    <addaction name="actionApply_Mask"/>
    <addaction name="actionPeak_Areas"/>
    <addaction name="actionMultiply_by_Vector"/>
    <addaction name="actionNormalization"/>
    <addaction name="actionStandardize_Data"/>
    <addaction name="actionDimensionality_Reduction"/>
    <addaction name="actionSpectral_Derivative"/>
    <addaction name="actionWavelength_Shift"/>
    <addaction name="actionCalibration_Transfer"/>
    <addaction name="actionCalibration_Transfer_CV"/>
   </widget>
   <widget class="QMenu" name="menuRegression">
    <property name="title">
     <string>Regression</string>
    </property>
    <addaction name="actionCross_Validation"/>
    <addaction name="actionTrain"/>
    <addaction name="actionPredict"/>
    <addaction name="actionLocal_Regression"/>
    <addaction name="actionSubmodel_Blend"/>
    <addaction name="actionSave_Trained_Model"/>
    <addaction name="actionRestore_Trained_Model"/>
   </widget>
   <widget class="QMenu" name="menuVisualization">
    <property name="title">
     <string>Visualization</string>
    </property>
    <addaction name="actionPlot"/>
    <addaction name="actionPlot_Spectra"/>
    <addaction name="actionPlot_ICA_PCA"/>
   </widget>
   <widget class="QMenu" name="menuClassification">
    <property name="title">
     <string>Classification</string>
    </property>
    <addaction name="actionSupervised"/>
    <addaction name="actionCluster"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionLoad_Data"/>
    <addaction name="actionSave_Current_Data"/>
    <addaction name="actionRead_ChemCam_Data"/>
    <addaction name="actionRename_Data"/>
    <addaction name="actionLook_Up_Metadata"/>
    <addaction name="actionRemove_Rows"/>
    <addaction name="actionCombine_Data_Sets"/>
    <addaction name="actionOutlier_Removal"/>
    <addaction name="actionSplit_Data"/>
    <addaction name="actionStratified_Folds"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuData"/>
   <addaction name="menuPreprocessing"/>
   <addaction name="menuClassification"/>
   <addaction name="menuRegression"/>
   <addaction name="menuVisualization"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRead_ChemCam_Data">
   <property name="text">
    <string>Load ChemCam Data</string>
   </property>
  </action>
  <action name="actionLoad_Data">
   <property name="text">
    <string>Load CSV Data</string>
   </property>
  </action>
  <action name="actionSet_Output_Path">
   <property name="text">
    <string>Set Output Path</string>
   </property>
  </action>
  <action name="actionSave_Current_Data">
   <property name="text">
    <string>Save Data to CSV</string>
   </property>
  </action>
  <action name="actionCreate_New_Workflow">
   <property name="text">
    <string>New Window</string>
   </property>
  </action>
  <action name="actionRestore_Workflow">
   <property name="text">
    <string>Restore Workflow</string>
   </property>
  </action>
  <action name="actionSave_Current_Workflow">
   <property name="text">
    <string>Save Workflow</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRemove_Rows">
   <property name="text">
    <string>Remove Rows</string>
   </property>
  </action>
  <action name="actionSplit_Data">
   <property name="text">
    <string>Split Data</string>
   </property>
  </action>
  <action name="actionResample">
   <property name="text">
    <string>Resample</string>
   </property>
  </action>
  <action name="actionRemove_Baseline">
   <property name="text">
    <string>Remove Baseline</string>
   </property>
  </action>
  <action name="actionApply_Mask">
   <property name="text">
    <string>Apply Mask</string>
   </property>
  </action>
  <action name="actionPeak_Areas">
   <property name="text">
    <string>Peak Areas</string>
   </property>
  </action>
  <action name="actionMultiply_by_Vector">
   <property name="text">
    <string>Multiply by Vector</string>
   </property>
  </action>
  <action name="actionNormalization">
   <property name="text">
    <string>Normalize</string>
   </property>
  </action>
  <action name="actionDimensionality_Reduction">
   <property name="text">
    <string>Dimensionality Reduction</string>
   </property>
  </action>
  <action name="actionStratified_Folds">
   <property name="text">
    <string>Stratified Folds</string>
   </property>
  </action>
  <action name="actionCross_Validation">
   <property name="text">
    <string>Cross Validation</string>
   </property>
  </action>
  <action name="actionTrain">
   <property name="text">
    <string>Regression Train</string>
   </property>
  </action>
  <action name="actionSubmodel_Blend">
   <property name="text">
    <string>Blend Submodel Predictions</string>
   </property>
  </action>
  <action name="actionPredict">
   <property name="text">
    <string>Regression Predict</string>
   </property>
  </action>
  <action name="actionPlot">
   <property name="text">
    <string>Plot</string>
   </property>
  </action>
  <action name="actionPlot_Spectra">
   <property name="text">
    <string>Plot Spectra</string>
   </property>
  </action>
  <action name="actionPlot_ICA_PCA">
   <property name="text">
    <string>Plot ICA/PCA</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionOpen_Workflow">
   <property name="text">
    <string>Open Workflow</string>
   </property>
  </action>
  <action name="actionOn_2">
   <property name="text">
    <string>On</string>
   </property>
  </action>
  <action name="actionOff_2">
   <property name="text">
    <string>Off</string>
   </property>
  </action>
  <action name="actionRename_Data">
   <property name="text">
    <string>Rename Data</string>
   </property>
  </action>
  <action name="actionQtmodern">
   <property name="text">
    <string>qtmodern</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionOn">
   <property name="text">
    <string>On</string>
   </property>
  </action>
  <action name="actionOff">
   <property name="text">
    <string>Off</string>
   </property>
  </action>
  <action name="actionBrace_yourself">
   <property name="text">
    <string>Brace yourself</string>
   </property>
  </action>
  <action name="actionClear_Workflow">
   <property name="text">
    <string>Clear Workflow</string>
   </property>
  </action>
  <action name="actionSpectral_Derivative">
   <property name="text">
    <string>Spectral Derivative</string>
   </property>
  </action>
  <action name="actionData_Box">
   <property name="text">
    <string>Data Box</string>
   </property>
  </action>
  <action name="actionCombine_Data_Sets">
   <property name="text">
    <string>Combine Data Sets</string>
   </property>
  </action>
  <action name="actionOutlier_Removal">
   <property name="text">
    <string>Remove Outliers</string>
   </property>
  </action>
  <action name="actionSave_Trained_Model">
   <property name="text">
    <string>Save Trained Model</string>
   </property>
  </action>
  <action name="actionRestore_Trained_Model">
   <property name="text">
    <string>Restore Trained Model</string>
   </property>
  </action>
  <action name="actionSupervised">
   <property name="text">
    <string>Supervised</string>
   </property>
  </action>
  <action name="actionK_means">
   <property name="text">
    <string>K-means</string>
   </property>
  </action>
  <action name="actionCluster">
   <property name="text">
    <string>Clustering</string>
   </property>
  </action>
  <action name="actionLook_Up_Metadata">
   <property name="text">
    <string>Look Up Metadata</string>
   </property>
  </action>
  <action name="actionLocal_Regression">
   <property name="text">
    <string>Local Regression</string>
   </property>
  </action>
  <action name="actionWavelength_Shift">
   <property name="text">
    <string>Wavelength Shift</string>
   </property>
  </action>
  <action name="actionPlot_testmodule">
   <property name="text">
    <string>plot_testmodule</string>
   </property>
  </action>
  <action name="actionCalibration_Transfer">
   <property name="text">
    <string>Calibration Transfer</string>
   </property>
  </action>
  <action name="actionCalibration_Transfer_CV">
   <property name="text">
    <string>Calibration Transfer CV</string>
   </property>
  </action>
  <action name="actionSave_Regression_Model">
   <property name="text">
    <string>Save Regression Model</string>
   </property>
  </action>
  <action name="actionRestore_Regression_Model">
   <property name="text">
    <string>Restore Regression Model</string>
   </property>
  </action>
  <action name="actionStandardize_Data">
   <property name="text">
    <string>Standardize</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
